Seasonizer/CanvasViewController.swift

```diff
@@ -130,7 +132,9 @@ class CanvasViewController: UIViewController, UINavigationControllerDelegate, UI
     
     
     // TODO: Implement an `@IBAction func unwindToCanvas(segue: UIStoryboardSegue)` Unwing Segue that the `AccessoryListViewController` can exit to.
-    
+    @IBAction func unwindToCanvas(segue: UIStoryboardSegue){
+        dismissViewControllerAnimated(true, completion: nil)
+    }
```

Comment:
Genau, wie @ChrisBtt sagt wird durch so eine Unwind Segue einfach der Punkt in der View Controller Hierarchie definiert, zu der die Segue zurückkehrt. Das passiert dann automatisch. Du musst nur eine Verbindung zur `Exit` Schaltfläche ziehen und der Segue dann einen `identifier` geben:
<img width="331" alt="bildschirmfoto 2016-05-21 um 13 37 35" src="https://cloud.githubusercontent.com/assets/746230/15448142/40e24046-1f59-11e6-8d69-d8e93ff65dd7.png">

Dann kannst du in `unwindToCanvas` und `prepareForSegue` je nach `identifier` entscheiden, was zu tun ist.